lib/rules/attributes-order.js

```diff
@@ -23,6 +23,8 @@ function getAttributeType (name, isDirective) {
       return 'EVENTS'
     } else if (name === 'html' || name === 'text') {
       return 'CONTENT'
+    } else {
+      return 'OTHER_DIRECTIVE'
```

Comment:
At the moment this category is not defined, so custom directives are always required to go first.